cypress/support/ui/common.js

```diff
@@ -34,3 +34,26 @@ Cypress.Commands.add('checkPopUpMessage', (message) => {
       element.next('button').click();
     });
 });
+
+Cypress.Commands.add('getTable', { prevSubject: true }, (subj, options = {}) => {
+  // eslint-disable-next-line no-magic-numbers
+  if (subj.get().length > 1) throw new Error(`Selector "${subj.selector}" returned more than 1 element.`);
```

Comment:
```suggestion
```suggestion
  if (subj.get().length) {
    throw new Error(`Selector "${subj.selector}" returned more than 1 element.`);
  }
```
```